app/graphql/enums/batch_import_status_enum.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class Enums::BatchImportStatusEnum < Types::BaseEnum
-  BatchImport.statuses.each do |value_string, _value_number|
-    value value_string, value_string
+  BatchImport::STATUSES.each do |status|
```

Comment:
I'm not sure how I feel about using the STATUSES constant instead of BatchImport.statuses.  It feels one step removed from what we're trying to do here in the enum (which is to have a 1:1 relationship with the values in BatchImport.statuses).

Reply:
BatchImport.statuses doesn't exist anymore though, I am confused.

Reply:
And rightfully so!  I am rescinding this comment.